src/components/PeopleFilters.tsx

```diff
@@ -29,40 +113,60 @@ export const PeopleFilters = () => {
           <div className="level-left">
             <a
```

Comment:
Errors from the linter must be fixed
<img width="1385" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/54860641/bc1843b1-7d87-4c68-98b3-999a4257a124">